cli/index.ts

```diff
@@ -11,6 +12,9 @@ program
 	.name( 'studio' )
 	.description( 'Studio by WordPress.com CLI' )
 	.version( version )
+	.hook( 'preAction', () =>
+		bumpAggregatedUniqueStat( StatsGroup.STUDIO_CLI_USAGE_UNIQUE, StatsMetric.SUCCESS, 'weekly' )
+	)
```

Comment:
```suggestion
	.hook( 'preAction', () => {
		bumpAggregatedUniqueStat( StatsGroup.STUDIO_CLI_USAGE_UNIQUE, StatsMetric.SUCCESS, 'weekly' )
	} )
```
Nit, but since we're not intentionally returning anything, I say let's wrap the function body in curly braces.